hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsClient.java

```diff
@@ -278,7 +282,8 @@ public AbfsRestOperation renamePath(final String source, final String destinatio
   }
 
   public AbfsRestOperation append(final String path, final long position, final byte[] buffer, final int offset,
-                                  final int length) throws AzureBlobFileSystemException {
+                                  final int length, boolean flush, boolean isClose)
```

Comment:
This method is supposed to be called from AbfsOutputStream.java -> writeCurrentBufferToService(), but there is no change for call to this API within that method. How did the compilation pass.

And, (probably a repeat question of config query posted above, still) can you please why add flush and close needs to be added to Append method.

Reply:
New iteration has the caller updated.